testing/dm_configs/cluster-kubeflow.yaml

```diff
@@ -23,15 +23,24 @@ resources:
     # Path for the bootstrapper image.
     bootstrapperImage: gcr.io/kubeflow-images-public/bootstrapper:latest
     bootstrapperConfig: |
+      registries:
+        # For e2e test target repo we skip registry repo and branch
```

Comment:
Why can't we just specify the registry as file URL in the case where we want to test the code in a presubmit/postsubmit? And then for the file URL specify the location of the source code on the volume.

Reply:
This comment is incorrect. I have updated this. In the workflow we build the bootstrap image and we bake the source code from the PR into the bootstrap image. The source code is obtained from the checked out git repo in the shared NFS.

The build_image step is [here](https://github.com/kubeflow/kubeflow/blob/master/testing/workflows/components/workflows.libsonnet#L444). The build_image.sh script is [here](https://github.com/kubeflow/kubeflow/blob/master/bootstrap/build_image.sh)